app/views/selfassessment/viewPayments.scala.html

```diff
@@ -0,0 +1,65 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.auth.requests.UserRequest
+@import config.ConfigDecorator
+@import util.LocalPartialRetriever
+@import uk.gov.hmrc.renderer.TemplateRenderer
+@import viewmodels.SelfAssessmentPayment
+
+
+@(payments: List[SelfAssessmentPayment])(implicit request: UserRequest[_], configDecorator: ConfigDecorator, partialRetriever: LocalPartialRetriever, templateRenderer: TemplateRenderer, messages: play.api.i18n.Messages)
+
+@main(messages("title.selfAssessment.viewPayments.h1"), Some(messages("label.your_personal_tax_account")), backlinkUrl = Some(controllers.routes.HomeController.index().url)) {
+    <div class="grid-row">
+        <div class="column-two-thirds">
+
+            <header class="page-heading">
+                <h1 class="heading-xlarge">
+                    @messages("title.selfAssessment.viewPayments.h1")
+                </h1>
+                <p class="heading-secondary"><span class="visuallyhidden">
+                    </span>@request.retrievedName.getOrElse(messages("label.your_account"))
+            </header>
+            @if(payments.isEmpty) {
+                <p>@messages("label.selfAssessment.noPaymentsIn60")</p>
+            } else {
+              <p>@messages("label.selfAssessment.balanceUpdateAdvisory")</p>
+              <table>
+              <thead>
+                <tr>
+                  <th scope="col">Date</th>
+                  <th scope="col">Reference number</th>
+                  <th scope="col" class="numeric">Amount</th>
```

Comment:
These need to be changed to use Messages API once Welsh comes back

Reply:
Oops, good catch!